muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/schooldatalocal/LocalUserSchoolDataBridge.java

```diff
@@ -537,6 +538,11 @@ public String findUsername(String identifier) {
   }
 
   @Override
+  public List<GroupUser> listGroupUsersByGroupAndType(String groupIdentifier, GroupUserType type) {
+    // TODO Auto-generated method stub
```

Comment:
If changes are made to the pr, remove todo's also - we'll come back to the local bridge if we do but it's not imminent to do task (just clutters eclipse).